docs/Home.md

```diff
@@ -1,18 +1,71 @@
-# Welcome to Inductiva API
-
-Inductiva is a Python package for executing large-scale simulations of physical 
-systems directly in the cloud.
-
-Inductiva API offers distinct advantages:
-
-- 🔄 It consolidates various simulation domains, including fluid and molecular 
-dynamics, plasmas, and structural mechanics, under a single unified entry point.
-- 📦 Eliminates the need for installing and managing complex simulation software 
-and corresponding dependencies.
-- 🚀 Allows running hundreds or even thousands of simulations concurrently, with 
-no coding.
-- 💽 Automatically optimizes hardware configurations for each type of simulation 
-(e.g., CPU vs. GPU, appropriate number of CPU cores, RAM, etc.).
-- 🐍 With Inductiva API, you are not limited to a pre-defined GUI or intricate 
-configuration languages and scripts. Instead, you write small Python programs 
-that seamlessly integrate with your existing codebase and ML framework.
+![Inductiva_Logo_noBG(259 x 76 px)](https://github.com/inductiva/inductiva/assets/102880975/c67e31eb-2775-4ee5-ac64-027f4591c0cc)
+# Inductiva: a Python package for running large-scale simulations of physical phenomena on the Cloud.
+
+Inductiva is a Python package designed for researchers, scientists, and engineers 
+focused on leveraging the full potential of [free and open-source physical simulation software packages](). 
```

Comment:
I think the ideas are here, but we may be able to make this even more compact, by dropping words that will become obvious later.

Inductiva is an API that allows scientists and engineers to fully exploit the potential of [open-source simulation software] and scale their simulation projects to hundreds of machines by merely writing a simple python script. 

Inductiva API abstracts way all the complexity of configuring, managing and optimizing costs of Cloud resources, and provides an easy-to-use Python interface to launch a variety of computational resources, including MPI clusters, and orchestrate simulation pipelines that use open-source simulation packages.  

wdyt: @sergiomsantos @hpenedones ?